tests/unittests/internal_docs_tests.py

```
@@ -0,0 +1,135 @@
+
+        self.assertFalse(is_hidden_from_docs(Visible))
+
+        @hide_from_docs
+        class Hidden:
+            pass
+
+        self.assertTrue(is_hidden_from_docs(Hidden))
+
+    def test_hides_instance_method(self):
+        class VisibleContainer:
+            def method(self):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `method`

Add return type annotation: `None`

(ANN202)